runtime-modules/content/src/tests/transfers.rs

```diff
@@ -232,158 +196,166 @@ fn accept_transfer_status_fails_with_non_channel_owner() {
 #[test]
 fn accept_transfer_status_fails_with_invalid_balance_for_members() {
     with_default_mock_builder(|| {
-        run_to_block(1);
-
-        create_initial_storage_buckets_helper();
-        increase_account_balance_helper(DEFAULT_MEMBER_ACCOUNT_ID, INITIAL_BALANCE);
-        create_default_member_owned_channel();
-
-        let price = INITIAL_BALANCE + 1; // higher than initial balance
+        ContentTest::with_member_channel().setup();
+        let balance_pre = Balances::<Test>::usable_balance(&DEFAULT_MEMBER_ACCOUNT_ID);
+        let price = balance_pre.saturating_add(DEFAULT_CHANNEL_TRANSFER_PRICE);
```

Comment:
I'm not sure why would we need to care about `DEFAULT_MEMBER_ACCOUNT_ID` balance here